pynq/lib/__init__.py

```diff
@@ -41,9 +41,6 @@
 from .rgbled import RGBLED
 from .switch import Switch
 from .button import Button
-from .pmod import *
-from .arduino import *
-from .intf import *
```

Comment:
This change is because we may have conflict definitions in different subpackages. For example, `arduino` subpackage and `intf` subpackage may both define `ARDUINO` constant.